### ISSUE TITLE
`@stylistic/indent`: multiline function call with operators in arguments

### ISSUE THREAD

**Issue by Tiuipuv** [eslint-stylistic/eslint-stylistic#753]
### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

When including operators in arguments of a function call that spans multiple lines, the CallExpression default does not behave as expected.

e.g. this is normal, with it wanting a default tab of 2 for all arguments on separate lines:
```js
console.log("sample",
  " without problem");
```
But when an argument value spans multiple lines like this:
```js
console.log("sample" + 
  " with",
  "problem");
```
The linter wants the spacing before "problem" to be removed.

### Reproduction

https://stackblitz.com/edit/github-1nkskuvv?file=src%2Findex.ts

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests
